validate/validate.go

```diff
@@ -128,6 +140,11 @@ func (v *Validator) CheckRootfsPath() (msgs []string) {
 		msgs = append(msgs, fmt.Sprintf("The root path %q is not a directory.", rootfsPath))
 	}
 
+	rootParent := filepath.Dir(absRootPath);
+	if rootParent == string(filepath.Separator) || rootParent != absBundlePath {
```

Comment:
You mean rootParent can't be "/"?  It seems there is no such limitation in SPEC.